lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -183,6 +183,9 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
 
             String govukSigninJourneyId = clientOAuthSessionItem.getGovukSigninJourneyId();
 
+            List<Vot> vot = clientOAuthSessionItem.getRequestedVotsByStrength();
+            Vot minimumRequestedVotsByStrength = vot.isEmpty() ? Vot.P2 : vot.get(0);
```

Comment:
Can we make this singular `minimumRequestedVotByStrength` as it's just a single VOT?